### ISSUE TITLE
Add edge attribute to allow treatment as loss by recursive methods

### ISSUE THREAD

**Issue by noracato** [quintel/atlas#179]
Edges should be allowed the following attribute 
```
- treat_as_loss = true
```

We need to allow a bool to be set for this optional attribute
